src/game/resource.c

```diff
@@ -199,6 +200,21 @@ resource_type resource_map_legacy_inventory(int id)
     return resource;
 }
 
+resource_type resource_produced_by_building_type(int building_type)
+{
+    for (int resource = 0; resource < RESOURCE_ALL; resource++) {
+        if (resource_info[resource].industry == building_type) {
+            return resource;
+        }
+    }
+    return RESOURCE_NONE;
+}
+
+int resource_production_per_month(resource_type resource)
```

Comment:
Suggestion: since you added this to a function, you can check for the climate here and return half the production for wheat.

I think this would be a great stepping stone to centralise the actual industry production in the future and not just the displayed efficiency.

Reply:
Good idea, just updated the code to move it.